css-scroll-snap-2/Overview.bs

```diff
@@ -137,93 +108,6 @@ Motivating Examples {#examples}
--------------------------------------------
-
-	<pre class="propdef shorthand">
-	Name: scroll-start
-	Value: [ auto | start | end | center | left | right | top | bottom | <<length-percentage [0,∞]>> ]{1,2}
-	</pre>
-
-	This property is a shorthand property that sets all of the scroll-start-* longhands in one declaration.
-	The first value defines the scroll starting point in the block axis,
-	the second sets it in the inline axis. If the second value is omitted, it defaults to ''scroll-start/start''. If ''scroll-start-target'' is set on any child, 'scroll-start' is not used, in favor of using the element as the offset.
-
-	Values are defined as follows:
-
-	<dl dfn-type=value dfn-for="scroll-start, scroll-start-x, scroll-start-y, scroll-start-block, scroll-start-inline">
-		<dt><dfn>auto</dfn>
-		<dd>
-			...
-
-		<dt><dfn><<length-percentage [0,∞]>></dfn>
-		<dd>
-			...
-
-			Negative values are invalid.
-			Values corresponding to a length greater than the width/height of the scrollport
-			are valid,
-			but clamped to the width/height of the scrollport.
-
-		<dt><dfn>start</dfn>
-		<dt><dfn>center</dfn>
-		<dt><dfn>end</dfn>
-		<dd>
-			Equivalent to ''0%'', ''50%'', and ''100%'', respectively.
-	</dl>
-
-<h4 id="display-none-behavior">
-Interaction with ''display: none'' and initial creation</h4>
-
-	Same behavior that animations follow with [[#first-layout]].
-
-<h4 id="slow-page-load-behavior">
-Slow page loading or document streaming behavior</h4>
-
-	TODO
-
-<h4 id="fragment-navigation-behavior">
-Interaction with "fragment navigation"</h4>
-
-	TODO
-
-	If the scrollport has a in-page '':target'' via a URL fragment or a previous scroll position, then 'scroll-start' is unused. Existing target logic should go unchanged. This makes 'scroll-start' a soft request in the scroll position resolution routines.
-
-<h4 id="place-content-behavior">
-Interaction with 'place-content'</h4>
-
-	TODO
-
-	Note: While 'place-content' can make a scroller appear to start in the center
-	or end, no browser supports it and it appears complicated to resolve.
-
-<h4 id="find-in-page-behavior">
-Interaction with "find in page"</h4>
-
-	TODO
-
-<h4 id="scroll-snap-container-behavior">
-Interaction with "scroll-snap" containers</h4>
-
-	This effectively will layout and start scroll at the snapped child, thus negating / cancelling 'scroll-start'. 'scroll-start' will only work if nothing else has effected the scroll position.
-
-<h4 id="nested-scrollers">
-Nested scrollers with 'scroll-start'</h4>
-
-	Should follow patterns that scroll snap has set.
-
-<h4 id="toggling-display-none">
-Interaction when 'display' is toggled</h4>
-
-	Same behavior that animations follow with [[#first-layout]].
-
-<h4 id="rtl">
-Interaction with RTL and LTR</h4>
-
-	Logical properties are offered for length offsets that should be flow relative. Furthermore, the ''end'' and ''start'' keywords are always logical.
-
 The 'scroll-start-target' property {#scroll-start-target}
```

Comment:
scroll-start-target says it is a shorthand but I think that's no longer true since becoming a single property, right?

Reply:
Yes that's correct. Good catch.

Reply:
Remnants of `scroll-start-target-*` longhands are still [there](https://github.com/w3c/csswg-drafts/blob/0ac5d9da9e408afe237aae6ecdd8e4cbc6741fd6/css-scroll-snap-2/Overview.bs#L176).

Reply:
Thanks for pointing this out. Created https://github.com/w3c/csswg-drafts/pull/10554 to remove them.